templates/observables/br_d_l_to_d_l.cpp.in

```diff
@@ -0,0 +1,267 @@
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+/**
+ * @file @ModelName@_@filename@.cpp
+ *
+ * This file was generated at @DateAndTime@ with FlexibleSUSY
+ * @FlexibleSUSYVersion@ and SARAH @SARAHVersion@
+ */
+
+#include <valarray>
+#include <complex>
+#include <fstream>
+#include <iomanip>
+
+#include "@ModelName@_mass_eigenstates.hpp"
+#include "cxx_qft/@ModelName@_qft.hpp"
+
+#include "@ModelName@_@filename@.hpp"
+#include "@ModelName@_FFV_form_factors.hpp"
+#include "json.hpp"
+#include "wrappers.hpp"
+
+@npf_headers@
+
+namespace flexiblesusy {
+
+namespace @ModelName@_cxx_diagrams {
+namespace npointfunctions {
+
+@npf_definitions@
+
+} // namespace npointfunctions
+} // namespace @ModelName@_cxx_diagrams
+
+namespace {
+
+std::valarray<std::complex<double>> zero(
+   int generationIndex1,
+   int generationIndex2,
+   const @ModelName@_mass_eigenstates&,
+   bool){
+   std::valarray<std::complex<double>> res {0.0, 0.0, 0.0, 0.0};
+   return res;
+}
+
+/**
+ * @brief Writes Wilson coefficients in a WCxf format [1712.05298] for
+ *        WET [https://wcxf.github.io/assets/pdf/WET.flavio.pdf].
+ * @tparam Down Type of a down quark.
+ */
+template <class Down>
+void write_btosmumu(
+   const Eigen::Array<std::complex<double>, 10, 1>& coeffs,
+   const @ModelName@_cxx_diagrams::context_base& context,
+   const softsusy::QedQcd& qedqcd) {
+
+   const auto complex_CKM = qedqcd.get_complex_ckm();
+
+   const std::complex<double> Vtb (complex_CKM(2, 2));
+   const std::complex<double> Vts (complex_CKM(2, 1));
+   const auto GF = qedqcd.displayFermiConstant();
+   const auto normalization = 4.*GF/Sqrt(2.) * Vtb*Conj(Vts) *
+      Sqr(unit_charge(context))*oneOver16PiSqr;
+
+   if (is_zero(std::abs(normalization))) {
+      ERROR("btosmumu: Normalization factor is vanishing.");
+      exit(EXIT_FAILURE);
+   }
+
+   const auto mb = context.mass<Down>({2});
+
+   // Dear reader, be aware of vector bosons (in loops) for b -> s mu mu.
+   // Some scheme dependence might be important.
+
+   // Compare to the basis in <DLDL/main.m>.
+   // They put SL on the first positions, we put BL.
+
+   // Here is the place for C7_bs = -1/2 A2R / normalization
+   // Here is the place for C7p_bs = -1/2 A2L / normalization
+
+   const auto C9_bsmumu   = (coeffs[5] + coeffs[4])/2. / normalization;
```

Comment:
Some minor remark: Floating point division is relatively "expensive". If the calculation takes too long, writing `.../(2*normalization*mb)` etc. would be better.

Reply:
Done in 3974d22668932756f8d95438d62ee077fb0c6100.